worlds/ahit/Regions.py

```diff
@@ -0,0 +1,947 @@
+    "The Subcon Well":              "subcon_cave",
+    "Toilet of Doom":               "chapter2_toiletboss",
+    "Queen Vanessa's Manor":        "vanessa_manor_attic",
+    "Mail Delivery Service":        "subcon_maildelivery",
+    "Your Contract has Expired":    "snatcher_boss",
+    "Time Rift - Sleepy Subcon":    "TimeRift_Cave_Raccoon",
+    "Time Rift - Pipe":             "TimeRift_Water_Subcon_Hookshot",
+    "Time Rift - Village":          "TimeRift_Water_Subcon_Dwellers",
+
+    "Alpine Free Roam":                 "AlpineFreeRoam",  # not an actual Time Piece
+    "The Illness has Spread":           "AlpineSkyline_Finale",
+    "Time Rift - Alpine Skyline":       "TimeRift_Cave_Alps",
+    "Time Rift - The Twilight Bell":    "TimeRift_Water_Alp_Goats",
+    "Time Rift - Curly Tail Trail":     "TimeRift_Water_AlpineSkyline_Cats",
+
+    "The Finale":                       "TheFinale_FinalBoss",
+    "Time Rift - Tour":                 "TimeRift_Cave_Tour",
+
+    "Bon Voyage!":                 "Cruise_Boarding",
+    "Ship Shape":                  "Cruise_Working",
+    "Rock the Boat":               "Cruise_Sinking",
+    "Time Rift - Balcony":         "Cruise_WaterRift_Slide",
+    "Time Rift - Deep Sea":        "Cruise_CaveRift_Aquarium",
+
+    "Nyakuza Free Roam":            "MetroFreeRoam",  # not an actual Time Piece
+    "Rush Hour":                    "Metro_Escape",
+    "Time Rift - Rumbi Factory":    "Metro_CaveRift_RumbiFactory"
+}
+
+# Guarantee that the first level a player can access is a location dense area beatable with no items
+guaranteed_first_acts = [
+    "Welcome to Mafia Town",
+    "Barrel Battle",
+    "She Came from Outer Space",
+    "Down with the Mafia!",
+    "Heating Up Mafia Town",  # Removed in umbrella logic
+    "The Golden Vault",
+
+    "Contractual Obligations",  # Removed in painting logic
+    "Queen Vanessa's Manor",  # Removed in umbrella/painting logic
+]
+
+purple_time_rifts = [
+    "Time Rift - Mafia of Cooks",
+    "Time Rift - Dead Bird Studio",
+    "Time Rift - Sleepy Subcon",
+    "Time Rift - Alpine Skyline",
+    "Time Rift - Deep Sea",
+    "Time Rift - Tour",
+    "Time Rift - Rumbi Factory",
+]
+
+chapter_finales = [
+    "Dead Bird Studio Basement",
+    "Your Contract has Expired",
+    "The Illness has Spread",
+    "Rock the Boat",
+    "Rush Hour",
+]
+
+# Acts blacklisted in act shuffle
+# entrance: region
+blacklisted_acts = {
+    "Battle of the Birds - Finale A":   "Award Ceremony",
+}
+
+# Blacklisted act shuffle combinations to help prevent impossible layouts. Mostly for free roam acts.
+blacklisted_combos = {
+    "The Illness has Spread":           ["Nyakuza Free Roam", "Alpine Free Roam", "Contractual Obligations"],
+    "Rush Hour":                        ["Nyakuza Free Roam", "Alpine Free Roam", "Contractual Obligations"],
+    "Time Rift - The Owl Express":      ["Alpine Free Roam", "Nyakuza Free Roam", "Bon Voyage!",
```

Comment:
Since Owl Express only has once entrance (Murder on the Owl Express) if Murder on the Owl Express gets randomized onto a Time Rift in a level, and the Time Rift in that level is only accessible through one act (in this case, the Deep Sea time rift only being accessible through Bon Voyage) and then the level required to enter that Time Rift gets mapped to Owl Express, then it's literally impossible to access because you would need to enter Owl Express to access the level to enter the Time Rift containing Murder on the Owl Express, which obviously makes no sense.

Reply:
Wow. That makes sense but wow lol. Can we mention this in a comment?
```
# Bon Voyage is here to prevent the cycle: Owl Express TR -> Bon Voyage -> Deep Sea TR -> Owl Express -> Owl Express TR
# which would make them all inaccessible since those TRs have no other entrances
```